src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+public class Ball {
+
+    private Color color;
+    private int number;
+
+    public Ball() {
+    }
+
+    public Ball(Color color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
+    public void setColor(Color color) {
+        this.color = color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
```

Comment:
if you dont use it - feel free to delete it
```suggestion
    public int getNumber() {
        return number;
    }

```